code/lib/cli/src/sandbox-templates.ts

```diff
@@ -73,7 +73,7 @@ export type Template = {
 const baseTemplates = {
   'cra/default-js': {
     name: 'Create React App (Javascript)',
-    script: 'npx create-react-app {{beforeDir}}',
+    script: 'npx create-react-app@latest {{beforeDir}}',
```

Comment:
Why is this needed? your PR doesn't say.
What is the issue? Is our CI testing/running old versions currently?

Reply:
Sorry Norbert. This is a draft PR. Forget to set the label